configure.ml

```diff
@@ -337,6 +337,8 @@ let args_options = Arg.align [
     " Compiles only bytecode version of Coq";
   "-debug", Arg.Set Prefs.debug,
     " Add debugging information in the Coq executables";
```

Comment:
What is the reason for keeping the `-debug` option since this is becoming the default anyways? Is it for compatibility for scripts which call `./configure -debug`?